refinery/data_set_manager/tests.py

```diff
@@ -1637,7 +1637,14 @@ def test_get_basic_node(self):
 class NodeIndexTests(APITestCase):
 
     def setUp(self):
+        data_set = DataSet.objects.create()
```

Comment:
Maybe... though I'd need to then dig though the structure to get the objects I really need: This is a test about nodes down at the bottom, than datasets up at the top.

Maybe create_dataset_with_necessary_models could be changed to return a dict providing access to the internal objects? @scottx611x : Ok if I merge without this and maybe loop back later?